package.json

```diff
@@ -1,56 +1,65 @@
 {
   "name": "nteract.io",
-  "version": "1.0.0",
   "description": "The server for nteract.io",
-  "scripts": {
-    "dev": "next",
-    "build": "next build",
-    "flow": "flow",
-    "start": "next start",
-    "prettier": "prettier --write './**/*.{js,scss}'"
-  },
-  "lint-staged": {
-    "{lib,spec}/**/*.{js,scss}": [
-      "prettier --write",
-      "git add"
-    ]
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/nteract/nteract.io.git"
-  },
+  "version": "1.0.0",
```

Comment:
Do we want to bump this in the refactor?

Reply:
No, I don't know who put that there. Thanks.

Reply:
@willingc totally misread the question and answered incorrectly. Sorry about that. I will bump the version to 2.0 I guess? Don't really know what we want to consider this.